dot/core/messages_integration_test.go

```diff
@@ -77,6 +80,10 @@ func TestService_HandleBlockProduced(t *testing.T) {
 		Keystore: keystore.NewGlobalKeystore(),
 	}
 
+	digestHandler := NewMockDigestHandler(ctrl)
+	digestHandler.EXPECT().HandleDigests(gomock.AssignableToTypeOf(new(types.Header)))
```

Comment:
yeah I think this would be to much since it has to be passed as part of the config in generating the service but the parent hash of the headeris retrieved through the service.